adapters/Pangle/PangleAdapter/Bidding/GADPangleRTBBannerRenderer.h

```diff
@@ -0,0 +1,24 @@
+//
+//  GADPangleRTBBannerRenderer.h
+//  PangleAdapter
+//
+//  Created by bytedance on 2022/1/11.
+//
+
+#import <Foundation/Foundation.h>
+
+#import <GoogleMobileAds/GoogleMobileAds.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface GADPangleRTBBannerRenderer : NSObject<GADMediationBannerAd>
+
+/// Asks the receiver to render the ad configuration.
+- (void)renderBannerForAdConfiguration:(nonnull GADMediationBannerAdConfiguration *)adConfiguration
+                   completionHandler:
+                       (nonnull GADMediationBannerLoadCompletionHandler)completionHandler;
+
+@end
+
+NS_ASSUME_NONNULL_END
```

Comment:
This can be removed.

Reply:
remove "NS_ASSUME_NONNULL_END"?

Reply:
Ditto, that's correct.

Reply:
Do I need remove all files or the current file ?